Formula/phive.rb

```diff
@@ -0,0 +1,18 @@
+class Phive < Formula
+  desc "The Phar Installation and Verification Environment (PHIVE)"
+  homepage "https://phar.io"
+  url "https://github.com/phar-io/phive/releases/download/0.12.1/phive-0.12.1.phar"
```

Comment:
This is marked as a pre-release. Homebrew only allows software that is marked stable.